### ISSUE TITLE
Feature request: Excluding files from report

### ISSUE THREAD

**Issue by jarrodrobins** [Madrapps/jacoco-report#3]
Hey, thanks for writing this action, it's great! 

I have a use case where I want to _exclude_ certain files from the pull request coverage report. I have already done this on the Jacoco side of things when generating the coverage report and have removed files that end in names like "Fragment" or "View". This is working correctly, as they are not visible in the HTML report Jacoco itself creates.

However, when creating a pull request, this action still shows those files if they have been changed at all in the PR. I would like a way to remove them.

<img width="472" alt="Screen Shot 2021-05-28 at 10 48 38 am" src="https://user-images.githubusercontent.com/1313663/119913918-7c8aa400-bfa2-11eb-8a27-5164a80c0a42.png">

In the above image - even though I've changed the Fragment and View files, I don't want them showing up because they've been filtered out of the Jacoco report.

**Comment by thsaravana** [comment]
Hey @jarrodrobins, could you check if `CreatePasscodeFragment` is present or not in the jacoco's XML report? Ideally, the action compares the entries in the XML report with the PR's changed files.
We will test this out and get back.

**Comment by jarrodrobins** [comment]
Hi @thsaravana - I think this may have simply been a false alarm, apologies for wasting your time! 

After your message I went digging further into the report and realised I had two issues -
- My exclusions weren't smart enough: internal classes weren't being filtered out properly (onClickListeners, as an example). Basically by filtering out files with `$` in them sorted this one out. 
- I misread the output. The Jacoco report lists class names whereas your plugin lists _files_. So, files which contain multiple classes would not show up in the report but would show up on a pull request. For example, a view class which I excluded would still show up in the report if I had a Kotlin extension function on, eg, a model object. 

So actually there's nothing wrong here at all, just me. 😅 One possible improvement (or option) could be to display classes instead of files, like the Jacoco report does?
